views/index.ejs

```diff
@@ -67,79 +68,128 @@
   </nav>
 
   <div class="row">
-    <div class="col-md-12 m-body">
-
+    </div>
+</div>
       <div>
         <!-- Insert html view templates here -->
           <ui-view></ui-view>
 
+     
+    </div>
+  
+
+<!--_________________________________________________________________________-->
+
+  <script type="text/ng-template" id="/home.html">
+  <link rel="stylesheet"  type="text/css" href="style2.css">
```

Comment:
<div>
>          <!-- Insert html view templates here -->
>            <ui-view></ui-view>

This code is needed because the home page would not show up without
this custom directive.

However,I merged all the CSS in one file


On Thu, Jun 11, 2015 at 2:36 PM, Aniltilanthe <notifications@github.com>
wrote:

> In views/index.ejs
> <https://github.com/Aniltilanthe/MasterApp/pull/2#discussion_r32201903>:
>
> >        <div>
> >          <!-- Insert html view templates here -->
> >            <ui-view></ui-view>
> >
> > +
> > +    </div>
> > +
> > +
> > +<!--_________________________________________________________________________-->
> > +
> > +  <script type="text/ng-template" id="/home.html">
> > +  <link rel="stylesheet"  type="text/css" href="style2.css">
>
> If this needed ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Aniltilanthe/MasterApp/pull/2/files#r32201903>.
>